tests/forte/data/data_store_test.py

```diff
@@ -584,6 +584,66 @@ def test_add_annotation_raw(self):
             ],
         )
 
+    def test_add_audio_annotation_raw(self):
+        # test add Document entry
+        tid_recording: int = self.data_store.add_audio_annotation_raw(
+            "ft.onto.base_ontology.Recording", 1, 5
+        )
+        # test add Sentence entry
+        tid_audio_utterance: int = self.data_store.add_audio_annotation_raw(
+            "ft.onto.base_ontology.AudioUtterance", 5, 8
+        )
+        tid_utterance: int = self.data_store.add_audio_annotation_raw(
+            "ft.onto.base_ontology.Utterance", 5, 8
+        )
+        num_doc = len(
+            self.data_store._DataStore__elements[
+                "ft.onto.base_ontology.Recording"
+            ]
+        )
+        num_sent = len(
+            self.data_store._DataStore__elements[
+                "ft.onto.base_ontology.AudioUtterance"
+            ]
+        )
+        num_utterance = len(
+            self.data_store._DataStore__elements[
+                "ft.onto.base_ontology.Utterance"
+            ]
+        )
+        self.assertEqual(num_doc, 1)
+        self.assertEqual(num_sent, 1)
+        self.assertEqual(num_utterance, 1)
+        tid = 77
+        self.data_store.add_annotation_raw(
+            "ft.onto.base_ontology.Recording", 0, 1, tid
```

Comment:
`ft.onto.base_ontology.Recording` is a subclass of `AudioAnnotation` and maybe we should use `add_audio_annotation_raw` here.

Reply:
Fixed it.